tests/test_waypoints_to_commands.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test process
+"""
+import dronekit
+
+from modules.waypoints_to_commands import waypoints_to_commands, ACCEPT_RADIUS
+
+
+def test_waypoints_to_commands():
+    mock_waypoints_data = [(42.123, -73.456), (42.789, -73.987), (42.555, -73.321)]
+    mock_altitude_data = 100
```

Comment:
Mock, fake, and stub should be reserved for interface implementations. Use a different word (e.g. `expected_` , `input` ) or don't use any (e.g. waypoints_data).

The `_data` suffix may be unnecessary as well.